interview-questions.md

```diff
@@ -10,42 +10,46 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 Your answer:
 
-Researched answer:
+Researched answer: Hashes in Ruby consist of key:value pairs. It is similar to an array, yet different because the indexing is not static as it is with arrays. In hashes, indexing is done with arbitrary keys of an object type. You create a hash by using the arrow function to assign a key to a value key => value and separating each key value pair with a comma. The entire has should be enclosed in curly braces. 
+
```

Comment:
would suggest including the relationship of the datatype of the key and the use of the hash rocket